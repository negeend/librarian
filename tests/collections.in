SAVE COLLECTION file.csv
ADD COLLECTION nonexistant.csv
aDd cOlLeCtIOn nonexistant.csv
save collection file.csv
add collection empty.csv
ADD COLLECTION empty.csv
ADD COLLECTION books.csv
add collection books.csv
add collection books2.csv
add collection books3.csv
LIST ALL
list all long
NUMBER COPIES
save collection file.csv
add collection file.csv
exit