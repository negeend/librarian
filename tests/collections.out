user: No books in system.

user: No such collection.

user: No such collection.

user: No books in system.

user: No books have been added to the system.

user: No books have been added to the system.

user: 17 books successfully added.

user: No books have been added to the system.

user: 20 books successfully added.

user: 1 books successfully added.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Biggles in the Battle (W.E. Johns)
So Long and Thanks for all the Fish (Douglas Adams)
The Hobbit (J.R.R. Tolkien)
The Hobbit (J.R.R. Tolkien)
Best Unit Ever (INFO1113)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111112: The Hobbit (J.R.R. Tolkien, Fantasy)
Currently available.

111113: The Iliad (Homer, Poetry)
Currently available.

111114: The Odyssey (Homer, Poetry)
Currently available.

111115: The Hitchhiker's Guide to the Galaxy (Douglas Adams, Comedy)
Currently available.

111116: So Long and Thanks for all the Fish (Douglas Adams, Comedy)
Currently available.

111117: Batman Volume 1 (Bob Kane, Comic)
Currently available.

111118: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111119: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111120: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111121: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111122: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111123: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111124: Wasteland (T.S. Eliot, Poetry)
Currently available.

111125: Biggles in the Battle (W.E. Johns, War)
Currently available.

111126: Aeneid (Virgil, Epic)
Currently available.

111127: Halloween (Jamie Lee Curtis, Horror)
Currently available.

111128: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111129: The Hobbit (J.R.R. Tolkien, Fantasy)
Currently available.

111130: The Iliad (Homer, Poetry)
Currently available.

111131: The Odyssey (Homer, Poetry)
Currently available.

111132: The Hitchhiker's Guide to the Galaxy (Douglas Adams, Comedy)
Currently available.

111133: So Long and Thanks for all the Fish (Douglas Adams, Comedy)
Currently available.

111134: Batman Volume 1 (Bob Kane, Comic)
Currently available.

111135: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111136: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111137: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111138: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111139: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111140: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111141: Wasteland (T.S. Eliot, Poetry)
Currently available.

111142: Biggles in the Battle (W.E. Johns, War)
Currently available.

111143: Aeneid (Virgil, Epic)
Currently available.

111144: Biggles in the Battle (W.E. Johns, War)
Currently available.

111145: So Long and Thanks for all the Fish (Douglas Adams, Comedy)
Currently available.

111146: The Hobbit (J.R.R. Tolkien, Fantasy)
Currently available.

111147: The Hobbit (J.R.R. Tolkien, Fantasy)
Currently available.

111148: Best Unit Ever (INFO1113, Horror)
Currently available.

user: Aeneid (Virgil): 2
Batman Volume 1 (Bob Kane): 2
Best Unit Ever (INFO1113): 1
Biggles in the Battle (W.E. Johns): 3
Halloween (Jamie Lee Curtis): 1
Harry Potter 1 (J.K. Rowling): 2
Harry Potter 2 (J.K. Rowling): 2
Harry Potter 3 (J.K. Rowling): 2
Harry Potter 4 (J.K. Rowling): 2
Harry Potter 5 (J.K. Rowling): 2
Harry Potter 6 (J.K. Rowling): 2
So Long and Thanks for all the Fish (Douglas Adams): 3
The Hitchhiker's Guide to the Galaxy (Douglas Adams): 2
The Hobbit (J.R.R. Tolkien): 4
The Iliad (Homer): 2
The Odyssey (Homer): 2
To Kill a Mockingbird (Harper Lee): 2
Wasteland (T.S. Eliot): 2

user: Success.

user: No books have been added to the system.

user: Ending Library process.
